src/stories/profile-modal/_types.tsx

```diff
@@ -0,0 +1,24 @@
+export interface ProfileModalArgs {
+  tempSelectedItem?: string;
+  currentLanguage: Language;
+  csmContactInfos: UserInfos;
+  userInfos: UserInfos;
+}
+
+export enum ProfileModalViewsEnum {
```

Comment:
Anzichè creare tipi enum da esportare (per quanto sia giusto e pulito) scrivi direttamente le stringhe in or nell'interface, sennò si devono importare anche tutti gli enum ogni volta per usare il componente.

Questo vale anche per tutti gli altri enum di seguito

Reply:
Sistemato, ma non mi è chiaro il perché. Puoi approfondire?